definitions/marts/looker_studio_marts/support_tickets/ls_support_ticket_volumes.sqlx

```diff
@@ -0,0 +1,238 @@
+        issue_type: "ARRAY of issue types selected for this ticket_id. This combines all issue types from all Zendesk instances.",
+        tags: "Array of metatags given to each ticket in Zendesk (can be user, organisation or issue related).",
+        is_duplicate: "ARRAY of issue types selected for this ticket_id in Zendesk. This combines all issue types from all Zendesk instances.",
+        created_at: "Date and time of that this ticket was logged in Zendesk.",
+        solved_at: "Date and time this ticket was resolved in Zendesk.",
+        updated_at: "Timestamp of the last update to this ticket in Zendesk.",
+        status_updated_at: "Timestamp of the last update to this ticket's status in Zendesk.",
+        subject: "The subject of tickets migrated to the new instance of CPD. Any tickets created after December 2023 will not have a subject. This data has been provided by Shaun Osborne and Declan fisher to help classify historic support tickets who's data was lost as part of the migration to the new instance of Zendesk",
+        proportionate_assignment: "A boolean field which indicates if a ticket has been proportionatly assigned to ECTE or NPQ. Tickets marked as true have insufficient data to accurately classify the ticket."
+    }
+}
+WITH
+  cpd_prog_data AS (
+  SELECT
+    ticket_id,
+    DATE(created_at) AS created_at,
+    group_name,
+    status,
+    COALESCE(LOWER(service_name), 'n/a') AS service_name,
+    ARRAY(
+    SELECT
+      LOWER(elem)
+    FROM
+      UNNEST(issue_type) AS elem) AS issue_type,
+    ARRAY(
+    SELECT
+      LOWER(elem)
+    FROM
+      UNNEST(tags) AS elem) AS tags,
+    solved_at,
+    updated_at,
+    status_updated_at,
+    LOWER(stm.subject) AS subject
+  FROM
+    `ecf-bq.zendesk_support_ticket_data.support_ticket_details_anonymised` AS sta
+  LEFT JOIN
+    `ecf-bq.static_tables.support_ticket_details_migrated` AS stm
+  ON
+    sta.ticket_id = stm.destination_id
+  WHERE
+    group_name LIKE 'CPD%' ),
+  duplicate_tickets AS (
+  SELECT
+    DISTINCT ticket_id,
+    CASE WHEN'closed_by_merge' IN UNNEST(tags) AND status = 'closed' THEN TRUE
+      ELSE FALSE
+  END
+    AS is_duplicate,
+  FROM
+    cpd_prog_data ),
+  -- First classification layer: uses the group_name to assign classification
+  first_classification_layer AS (
+  SELECT
+    cpd_prog_data.*,
+    CASE
+    -- WHEN'closed_by_merge' IN UNNEST(tags) then 'Duplicate ticket'
+      WHEN group_name IN ('CPD - ECTE', 'CPD - 2nd Line- LP and DP Queries', 'CPD - 2nd Line: Appropriate Bodies & Regs', 'CPD - 2nd line - Withdrawing or reinstating ECTs/mentors', 'CPD - 3rd Line Support (Dfe admin)', 'CPD - ECF', 'CPD - ECF - Access to DfE service', 'CPD - ECF - ECT registration issues', 'CPD - ECF - Mentor registration issues', 'CPD - ECF - Partnerships', 'CPD - 2nd line ECF review (policy)', 'CPD - 2nd line- Unable to change cohort due to declaration errors on the service') THEN 'CPD - ECTE'

```

Comment:
It might be just me but I think I'd find it easier to read this list if each element were on a separate line? Also it might be easier to edit